model/yeast-GEM.txt

```diff
@@ -1152,7 +1152,7 @@ r_1680	ceramide-2 (C26)[g] <=> ceramide-2 (C26)[er] 		-1000.00	1000.00	  0.00
 r_1681	ceramide-4 (C26)[g] <=> ceramide-4 (C26)[er] 		-1000.00	1000.00	  0.00
 r_1682	oxygen[c] + S-adenosyl-L-methionine[c] + zymosterol[c] => ergosta-5,7,22,24(28)-tetraen-3beta-ol[c] + H+[c] + 2 H2O[c] + S-adenosyl-L-homocysteine[c] 		  0.00	1000.00	  0.00
 r_1683	choline[e] =>  		  0.00	1000.00	  0.00
-r_1684	choline[ce] <=> choline[c] 	YGL077C or YOR161C	-1000.00	1000.00	  0.00
```

Comment:
Even though this one was not shown evidence for choline transport, but this was added due to the sequence similarity as documented in [SGD](https://www.yeastgenome.org/locus/S000005687) and [Biocyc](https://biocyc.org/gene?orgid=YEAST&id=G3O-33678#). If there is no clear evidence for not transport, I suggest we keep the original grRules. @edkerk @hongzhonglu?